### PR TITLE
Initial commit for homework 1

### PR DESCRIPTION
**Pull request by nsavelyeva**
I wanted to write a unit test for Java HTTP servlet. Since doGet is a protected method, then having usual maven project structure I faced a problem of calling it - I was able to resolve this in 2 ways: with an inner class as suggested here https://stackoverflow.com/questions/34092705/how-do-i-use-mockito-to-mock-a-protected-method, and with Java reflection. 
But still my test failed. As turned out, this was because of JSP templates, i.e if doGet() simply returns the content, then this simple and short solution works: http://claudioaltamura.de/java/unit-testing-servlets-with-mockito. However, I wanted something like this: https://examples.javacodegeeks.com/core-java/junit/junit-test-case-example-for-web-application/ but could not get it work.
Finally, I have left this idea because I think when we start using Spring then writing unit tests will be different and even much easier.

### CODE REVIEW

File: database/src/main/java/by/itacademy/model/Car.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Car {
+
+    private Long id;
+    private String brand;
+
+    public String getBrand() {
```
Comment (DzianisMatveyenka) [inline database/src/main/java/by/itacademy/model/Car.java]: раз используешь ломбок, то и геттер писать уже ни к чему

File: database/src/main/java/by/itacademy/model/User.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class User {
+
+    private Long id;
+    private String name;
+
+    public String getName() {
```
Comment (DzianisMatveyenka) [inline database/src/main/java/by/itacademy/model/User.java]: то же самое и здесь, не нужен get

File: database/src/main/java/by/itacademy/model/Car.java (hunk 1 above)
Comment (nsavelyeva) [reply database/src/main/java/by/itacademy/model/Car.java]: Спасибо, точно!
Я геттер написала до импорта ломбока и потом забыла удалить геттер.
Поправила.

File: database/src/main/java/by/itacademy/model/User.java (hunk 2 above)
Comment (nsavelyeva) [reply database/src/main/java/by/itacademy/model/User.java]: Спасибо, тоже поправила.

**Pull request merged**